spec/uploaders/image_uploader_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'carrierwave/test/matchers'
+
+RSpec.describe ImageUploader, type: :model do
+  include CarrierWave::Test::Matchers
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+
+    before do
+      ImageUploader.enable_processing = true
+      @uploader = ImageUploader.new(course, :logo)
+
+      File.open(File.join(Rails.root,'/spec/fixtures/files/picture.jpg')) do |f|
+        @uploader.store!(f)
+      end
+    end
+
+    after do
+      ImageUploader.enable_processing = false
+      @uploader.remove!
+    end
+
+    context 'the thumb version' do
+      it "scales down a landscape image to be exactly 64 by 64 pixels" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.